### ISSUE TITLE
TypeScript Compiler does not show error if some method parameter is missing in implemented interface

### ISSUE THREAD

**Issue by proalphaDev** [microsoft/TypeScript#32228]
**Search Terms:**
interface missing parameter no error 

**Code**

```ts
interface Test {
  update(a: string, b: number): void;
}

class TestClass implements Test {
  update(a: string): void {}
}
```

**Expected behavior:**
ts shows an error since TestClass does not correctly implement interface Test (see missing parameter b)

**Actual behavior:**
ts does not show an error.

**Playground Link:**
https://www.typescriptlang.org/play/#code/JYOwLgpgTgZghgYwgAgCoQM5mQbwFDLICuADgCZyQAUcAXMllKAOYA0yARvSEQLYfQAlPQBuAe2BkA3HgC+ePAgA2cDBjSYwAYRVrkwXiSUReEcOvRZcBYuUoQa9Ri2HJxk3PNlA

**Comment by MartinJohns** [comment]
https://github.com/microsoft/TypeScript/wiki/FAQ#why-are-functions-with-fewer-parameters-assignable-to-functions-that-take-more-parameters
